### PR TITLE
[Messenger] Restored wildcard support in routing

### PR DESCRIPTION
**Pull request by yceruto**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #27081 
| License       | MIT
| Doc PR        | -

Also fixing some CS warnings and code structure.

### CODE REVIEW

**Comment by sroze** [comment]
As proven by this regression... we need to test it. Can you add a test
please? :)

**Comment by yceruto** [comment]
You caught me! :D ... Alright, I'm adding the tests.

**Comment by yceruto** [comment]
Tests added.

**Review by sroze — APPROVED** [review]

File: src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/messenger_routing.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+$container->loadFromExtension('framework', array(
+    'messenger' => array(
+        'routing' => array(
+            'Symfony\Component\Messenger\Tests\Fixtures\DummyMessage' => array('amqp'),
+            'Symfony\Component\Messenger\Tests\Fixtures\SecondMessage' => array('amqp', 'audit'),
```
[the remark this answers is not in the document]
Comment (yceruto) [reply src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Fixtures/php/messenger_routing.php]: Done.

**Comment by nicolas-grekas** [comment]
Thank you @yceruto.

**Pull request merged**